lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -265,17 +266,35 @@ ModuleDependencyVector ClangImporter::bridgeClangModuleDependencies(
       swiftArgs.push_back(IncludeTree);
     }
 
+    // Ensure we get the "real", vfs-remapped path to the modulemap
+    // becuase it is used as an explicit input for the resulting compilation
+    // task.
```

Comment:
Nit:
```suggestion
    // Ensure we get the external path to the modulemap because it is used as an
    // explicit input for the resulting compilation task.
```